provisioning/transport/mqtt/src/ProvisioningChannelHandlerAdapter.cs

```diff
@@ -328,7 +326,7 @@ private async Task VerifyPublishPacketTopicAsync(IChannelHandlerContext context,
                 {
                     if (statusCode >= HttpStatusCode.BadRequest)
                     {
-                        var errorDetails = JsonConvert.DeserializeObject<ProvisioningErrorDetailsMqtt>(jsonData);
+                        ProvisioningErrorDetailsMqtt errorDetails = JsonConvert.DeserializeObject<ProvisioningErrorDetailsMqtt>(jsonData);
```

Comment:
Do you need to specify type here as you're specifying it during deserialization?

Reply:
+1 from me. Var seems appropriate here, still

Reply:
sure, this was a VS recommended change, but I can that it isn't really relevant here as we are already specifying the type